tools/curations: Gradle task `generateLicenseClassifications` fails

Here's the output:

```
./gradlew generateLicenseClassifications
> Task :generateLicenseClassifications FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':generateLicenseClassifications'.
> com.fasterxml.jackson.module.kotlin.MissingKotlinParameterException: Instantiation of [simple type, class org.ossreviewtoolkit.tools.curations.License] value failed for JSON property yml due to missing (therefore NULL) value for creator parameter yml which is a non-nullable type
   at [Source: (URL); line: 13, column: 3] (through reference chain: java.util.ArrayList[0]->org.ossreviewtoolkit.tools.curations.License["yml"])

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org
```

--- a/tools/curations/buildSrc/src/main/kotlin/ScanCodeLicenseDbClassifications.kt
+++ b/tools/curations/buildSrc/src/main/kotlin/ScanCodeLicenseDbClassifications.kt
@@ -70,14 +70,14 @@ private data class License(
     val category: String,
     val json: String,
     val yaml: String,
-    val html: String,
-    val text: String
+    val html: String
 )
 
 private data class LicenseDetails(
     val key: String,
     val shortName: String,
     val name: String,
+    val text: String,
     val category: String,
     val owner: String,
     val homepageUrl: String? = null,

--- a/tools/curations/buildSrc/src/main/kotlin/ScanCodeLicenseDbClassifications.kt
+++ b/tools/curations/buildSrc/src/main/kotlin/ScanCodeLicenseDbClassifications.kt
@@ -69,7 +69,7 @@ private data class License(
     val isDeprecated: Boolean,
     val category: String,
     val json: String,
-    val yml: String,
+    val yaml: String,
     val html: String,
     val text: String
 )

--- a/tools/curations/buildSrc/src/main/kotlin/ScanCodeLicenseDbClassifications.kt
+++ b/tools/curations/buildSrc/src/main/kotlin/ScanCodeLicenseDbClassifications.kt
@@ -70,7 +70,8 @@ private data class License(
     val category: String,
     val json: String,
     val yaml: String,
-    val html: String
+    val html: String,
+    val license: String
 )
 
 private data class LicenseDetails(

